kernel/power/strmm_kernel_16x8_power8.S

```diff
@@ -12,7 +12,7 @@ the documentation and/or other materials provided with the
 distribution.
 3. Neither the name of the OpenBLAS project nor the names of
 its contributors may be used to endorse or promote products
-derived from this software without specific prior written permission.
+derived from this software without specific prior written permission.  
```

Comment:
Spurious spaces here...

Reply:
Thanks, probably introduced them by using the github web editor. Wish we were back at that level of "problems" though.